color-changer.js

```diff
@@ -0,0 +1,42 @@
+function getRandomUnusedHue(usedHues) {
+    let randomHue;
+    while(randomHue === undefined || usedHues[randomHue]) {
+        randomHue = Math.floor(Math.random()*360);
```

Comment:
It won't. I actually deliberately left out the + 1 just for readability purposes. If it was critical to include that color, I would, but I'm already excluding most colors due to how i handle lighting and saturation.